src/components/LabeledRadioButton/LabeledRadioButton.spec.jsx

```diff
@@ -0,0 +1,81 @@
+import _ from 'lodash';
+import assert from 'assert';
+import React from 'react';
+import SyntheticEvent from 'react/lib/SyntheticEvent';
+import sinon from 'sinon';
+import { mount, shallow } from 'enzyme';
+
+import describeWithDOM from '../../util/describe-with-dom';
+import { common } from '../../util/generic-tests';
+
+import LabeledRadioButton from './LabeledRadioButton';
+import RadioButton from '../RadioButton/RadioButton';
+
+SyntheticEvent;
+sinon;
+describeWithDOM;
+
+describe('LabeledRadioButton', () => {
+	common(LabeledRadioButton);
+
+	describe('props', () => {
+		describe('isDisabled', () => {
+			it('passes the value through to its `RadioButton` instance.', () => {
+				const wrapper = shallow(<LabeledRadioButton isDisabled={true} />);
+
+				assert.equal(wrapper.find(RadioButton).prop('isDisabled'), true);
+			});
+
+			it('defaults to `false`.', () => {
+				const wrapper = mount(<LabeledRadioButton />);
+
+				assert.equal(wrapper.prop('isDisabled'), false);
+			});
+		});
+
+		describe('isSelected', () => {
+			it('passes the value through to its `RadioButton` instance.', () => {
+				const wrapper = shallow(<LabeledRadioButton isSelected={true} />);
+
+				assert.equal(wrapper.find(RadioButton).prop('isSelected'), true);
+			});
+
+			it('defaults to `false`.', () => {
+				const wrapper = mount(<LabeledRadioButton />);
+
+				assert.equal(wrapper.prop('isSelected'), false);
+			});
+		});
+
+		describe('onSelect', () => {
+			it('passes the value through to its `RadioButton` instance.', () => {
+				const foo = () => null;
+				const wrapper = shallow(<LabeledRadioButton onSelect={foo} />);
+
+				assert.equal(wrapper.find(RadioButton).prop('onSelect'), foo);
+			});
+
+			it('defaults to the Lodash `noop` method.', () => {
+				const wrapper = mount(<LabeledRadioButton />);
+
+				assert.equal(wrapper.prop('onSelect'), _.noop);
+			});
+		});
+
+		describe('pass throughs', () => {
+			it('passes through all props not defined in `propTypes` to its `RadioButton` instance.', () => {
+				const wrapper = shallow(
+					<LabeledRadioButton className='wut' isDisabled={true} isSelected={true} style={{ fontWeight: 'bold' }} onSelect={_.noop}
+							foo={1} bar={2} baz={3} qux={4} quux={5} />
+				);
+				const radioButtonProps = _.keys(wrapper.find(RadioButton).props());
+
+				// It should pass `foo`, `bar`, `baz`, `qux`, and `quux` through
+				// to the `RadioButton` instance.
+				_.forEach(['foo', 'bar', 'baz', 'qux', 'quux'], (prop) => {
+					assert(_.includes(radioButtonProps, prop));
```

Comment:
instead of using `_.includes` on `_.keys` just use `_.has` to check if the `prop` is on the object